tokio/CHANGELOG.md

```diff
@@ -1,9 +1,25 @@
-# 0.2.13 (February 28, 2019)
+# 0.2.14 (April 1, 2020)
+
+### Fixes
+- rt: concurrency bug in scheduler (#2273).
+- rt: concurrency bug with shell runtime (#2333).
+- test-util: correct pause/resume of time (#2253).
+- time: `DelayQueue` correct wakeup after `insert` (#2285).
+
+### Added
+- io: impl `RawFd`, `AsRawHandle` for std io types (#2335).
+- rt: automatic cooperative task yielding (#2160, #2343, #2349).
```

Comment:
when the blog post goes live, should we link to it?

Reply:
I will link in the github release post.